src/app/routers/field.py

```diff
@@ -321,3 +322,34 @@ def put(self, field_id):
             raise BadRequest("Couldn't update")
 
         return Response(status=200)
+
+    @API.doc(
+        responses={
+            200: 'OK',
+            401: 'Unauthorized',
+            400: 'Bad Request',
+            403: 'User is not the field owner'
+        }, params={
+            'field_id': 'Field id'
+        }
+    )
+    @login_required
+    # pylint: disable=no-self-use
+    def delete(self, field_id):
+        """
+        Delete field
+
+        :param field_id:
+        :return:
+        """
+        field = FieldService.get_by_id(field_id=field_id)
+        if field is None:
+            raise BadRequest('Field does not exist')
+        if current_user.id != field.owner_id:
+            raise Forbidden('Forbidden. User is not the field owner')
+        delete = FieldService.delete(field_id=field_id)
```

Comment:
check if field is already contained within a form, if it is - deletion is forbidden